cost/google/unused_cloudsql_instances/google_unused_cloudsql_instances.pt

```diff
@@ -7,7 +7,7 @@ severity "low"
 category "Cost"
 default_frequency "daily"
 info(
-      version: "2.7",
+      version: "2.8",
```

Comment:
Should we consider scraping recommendations from this "Google Unused CloudSQL Instances" policy?

Reply:
Not scraping from this policy until it can provide accurate cost data.